scaling_performance/scaling_cluster_metrics.adoc

```diff
@@ -0,0 +1,147 @@
+overhead to ensure that the storage requirements do not exceed calculated value.
+
+If the `METRICS_DURATION` and `METRICS_RESOLUTION` values are kept at the
+default (`7` days and `15` seconds respectively), it is safe to plan Cassandra
+storage size requirements for week, as in the values above.
+
+[WARNING]
+====
+Because {product-title} metrics uses the Cassandra database as a datastore for
+metrics data, if `USE_PERSISTANT_STORAGE=true` is set during the metrics set up
+process, `PV` will be on top in the network storage, with NFS as the default.
+However, using network storage in combination with Cassandra is not recommended,
+as per the
+link:http://docs.datastax.com/en/archived/cassandra/1.2/cassandra/architecture/architecturePlanningAntiPatterns_c.html[Cassandra
+documentation].
+====
+
+
+[[cluster-metrics-scaling-openshift-metrics-pods]]
+== Scaling {product-title} Metrics Pods
+
+One set of metrics pods (Cassandra/Hawkular/Heapster) is able to monitor at
+least 10,000 pods.
+
+[CAUTION]
+====
+Pay attention to system load on nodes where {product-title} metrics pods run.
+Use that information to determine if it is necessary to scale out a number of
+{product-title} metrics pods and spread the load across multiple {product-title}
+nodes. Scaling {product-title} metrics heapster pods is not recommended.
+====
+
+[[cluster-metrics-scaling-pods-prereqs]]
+=== Prerequisites
+
+If persistent storage was used to deploy {product-title} metrics, then you must
+xref:../dev_guide/persistent_volumes.adoc#dev-guide-persistent-volumes[create a
+persistent volume (PV)] for the new Cassandra pod to use before you can scale
+out the number of {product-title} metrics Cassandra pods. However, if Cassandra
+was deployed with dynamically provisioned PVs, then this step is not necessary.
+
+[[cluster-metrics-scaling-pods-cassandra]]
+=== Scaling the Cassandra Components
+
+Cassandra nodes use persistent storage. Therefore, scaling up or down is not
+possible with replication controllers.
+
+Scaling a Cassandra cluster requires modifying the
+`openshift_metrics_cassandra_replicas` variable and re-running the
+xref:../install_config/cluster_metrics.adoc#deploying-the-metrics-components[deployment].
+By default, the Cassandra cluster is a single-node cluster.
+ifdef::openshift-origin[]
+To deploy more nodes, provision storage if `openshift_metrics_cassandra_replicas` equals `pv` and
+increase the `openshift_metrics_cassandra_replicas` value.
+endif::openshift-origin[]
+
+ifdef::openshift-enterprise[]
+To scale up the number of {product-title} metrics hawkular pods to two
+replicas, run:
+
+----
+# oc scale -n openshift-infra --replicas=2 rc hawkular-metrics
+----
+
+Alternatively, update your inventory file and re-run the
+xref:../install_config/cluster_metrics.adoc#deploying-the-metrics-components[deployment].
+endif::openshift-enterprise[]
+
+[NOTE]
+====
+If you add a new node to a Cassandra cluster, the data stored in the cluster
```

Comment:
"If you add or remove a node to a Cassandra cluster and remove the "The same happens" sentence.